### PR TITLE
Implemented entities

### PR DESCRIPTION
**Pull request by vlad12345123**

### CODE REVIEW

File: InternshipBE/ExaLearn.Shared/Enums/LevelType.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿namespace Shared.Enums
+{
+    public enum LevelType
```
Comment (romanstat) [inline InternshipBE/ExaLearn.Shared/Enums/LevelType.cs:3]: in the terms of reference 6 levels, check

File: InternshipBE/ExaLearn.Bl/Services/FileService.cs [hunk 2]
```diff
@@ -10,30 +9,35 @@
...
                 throw new ValidationException("File not found!");
 
             var fileType = Path.GetExtension(file.FileName);
             var isNessesaryFormat = AudioExtensions.AvailableAudutionExtensions.Contains(fileType.ToLower());
 
+
             if (!isNessesaryFormat)
-                throw new ValidationException("File format does not meet the requirements!");
+            {
+                var availableFormats = string.Join(", ", AudioExtensions.AvailableAudutionExtensions.ToArray());
```
Comment (romanstat) [inline InternshipBE/ExaLearn.Bl/Services/FileService.cs]: remove toarray()

File: InternshipBE/ExaLearn.Dal/Database/ExaLearnDbContext.cs [hunk 3]
```diff
@@ -5,16 +5,20 @@
...
-    public class ExaLearnDbContext : IdentityDbContext<ApplicationUser, IdentityRole<int>,int> 
-        {
         public ExaLearnDbContext(DbContextOptions<ExaLearnDbContext> options) : base(options) { }
 
-        public DbSet<FileEntry> FileEntry { get; set; }
+        public DbSet<AudioFile> FileEntry { get; set; }
 
         protected override void OnModelCreating(ModelBuilder builder)
         {
+            builder.Entity<Question>()
```
Comment (romanstat) [inline InternshipBE/ExaLearn.Dal/Database/ExaLearnDbContext.cs]: for what?

Comment (vlad12345123) [reply InternshipBE/ExaLearn.Dal/Database/ExaLearnDbContext.cs]: for fun

**Review by romanstat — APPROVED** [review]

**Review by ExadelSandbox — APPROVED** [review]

**Pull request merged**